include/rocksdb/cloud/cloud_env_options.h

```diff
@@ -361,14 +361,19 @@ class CloudEnvOptions {
   // CLOUDMANIFEST)
   std::string cookie_on_open;
 
-  // If true, and cookie is not empty, besides uploading CLOUDMANIFEST-cookie to
-  // s3, we also upload CLOUDMANIFEST (no cookie suffix) to s3. This makes sure
-  // that we can quickly rollback if something unexpected happens.
+  // Experimental option!
+  // The cookie we will switch to when we open the db and roll the new epoch.
   //
-  // TODO(wei): This is a temporary option. Once we gain more confidence about cookie we
-  // should remove it
-  // Default: true
-  bool upload_cloud_manifest_without_cookie_suffix;
+  // NOTE: This option will only take effect when db exists (i.e., not new db).
+  // If it's a new db, CLOUDMANIFEST file will be created with `cookie_on_open`.
+  //
+  // TODO(wei): This is a temporarly option to unblock leader/follower rollout,
```

Comment:
Why? I think this is absolutely the way it should work long term! It's a good feature. S3 doesn't have any fencing mechanism ("stop the old leader from writing"); the ONLY way to protect against some old process being slow and still writing is to write our own thing under a different name.

The problem is that we keep generating CLOUDMANIFEST files, so eventually we should clean the old ones.

Reply:
Also we have to support "having two leader running for short period of time" even when leader follower is fully rolled out because it is possible for that to happen due to network partition (old leader doesn't realize there is a new leader in town)